classification/metrics/compute_vessel_metrics.py

```diff
@@ -769,39 +659,80 @@ def confusion_matrix(results):
     return ConfusionMatrix(cm_raw, cm_normalized)
 
 
-def load_inferred(inference_path, extractors, whitelist):
+def load_inferred(inference_table, label_table, extractors):
     """Load inferred data and generate comparison data
 
     """
-    with gzip.GzipFile(inference_path) as f:
-        with nlj.open(f, json_lib='ujson') as src:
-            for row in src:
-                if whitelist is not None and row['mmsi'] not in whitelist:
-                    continue
-                # Parsing dates is expensive and all extractors use dates, so parse them
-                # once up front
-                row['start_time'] = _parse(row['start_time'])
-                #dateutil.parser.parse(row['start_time'])
-                for ext in extractors:
-                    ext.extract(row)
+    query = """
+
+    SELECT c.* except (vessel_id), vessel_id as id FROM 
+    `{}` b
```

Comment:
I would assign `b` and `c` better names, it is not clear without knowing what you are doing which tables you are querying.

Reply:
Done